<?xml version="1.0"?>

<launch>

  <!-- estimator node -->
  <node pkg="icl_multiple_stationary" type="estimator_main_node" name="estimator_main_node" output="screen">
  <!-- <node pkg="icl_multiple_stationary" type="estimator_main_node" name="estimator_main_node"> -->

	  <!-- forward camera -->

	  <param name="cameraName" value="camera"/>
	  <param name="image_roi_percent" value="0.2"/>
	  <param name="numberFeaturesToFindPerPart" value="25"/>
    <param name="blockSize" value="3"/>
    <param name="qualityLevel" value="0.1"/>

    <param name="minFeatureDisEdge" value="20"/>
	  <param name="minFeaturesDanger" value="50"/>
    <param name="patchRadius" value="50"/>
    <param name="partitionRows" value="1"/>
    <param name="partitionCols" value="1"/>

    <param name="closeLoop" value="true"/>
    <param name="minFeaturesBad" value="14"/>
    <param name="minDistance" value="150"/>
    <param name="partitionSide" value="1"/>
    <param name="numberFeaturesPerPartRow" value="3"/>
    <param name="numberFeaturesPerPartCol" value="5"/>
    <param name="patchSizeBase" value="100"/>
    <param name="checkSizeBase" value="125"/>

    <!-- min and max depths-->
    <param name="zmin" value="1.0"/>
    <param name="zmax" value="6.0"/>

	  <!-- cut off frequencies-->
	  <param name="fq" value="10.0"/>
    <param name="fp" value="1.5"/>
    <param name="ft" value="10.0"/>
    <param name="fn" value="0.5"/>
    <param name="fd" value="50.0"/>
    <param name="fG" value="15.0"/>

    <param name="pcbx" value="0.105638"/>
    <param name="pcby" value="0.0702751"/>
    <param name="pcbz" value="0.28"/>

    <!-- <param name="qcbw" value="0.722352"/>
    <param name="qcbx" value="-0.691162"/>
    <param name="qcby" value="0.0217651"/>
    <param name="qcbz" value="0.00289808"/> -->

    <param name="qcbw" value="0.7071"/>
    <param name="qcbx" value="-0.7071"/>
    <param name="qcby" value="0.0"/>
    <param name="qcbz" value="0.0"/>



    <!-- integration window -->
    <param name="tau" value="3.0"/>

    <!-- experiment details -->
    <param name="saveExp" value="false"/>
    <param name="expName" value="s16"/>

  </node>

</launch>
